### PR TITLE
Fixes #1272

### LINKED ISSUE #1272: App Crashes
**Summary:**

'Pay To' field accepts empty entries and the spinner is empty which is not good at all.
'Pay From' field accepts empty entries and the spinner is empty which is not good at all.

App crashes,
On following actions:
Transfer->Transfer->Continue->Continue->App Crashed

**Expected behaviour:**

Give an error after pressing continue if the field was empty and if it succeeds,
Ask for Amount

**Observed behaviour:**

Desktop_20200118233021

**Device and Android version:**

One Plus 6
Android Version- Android Q

I would like to work on this!

### PR DESCRIPTION
**Pull request by divyank00**
Fixes #1272 

Please Add Screenshots If there are any UI changes.
Desktop_20200119190010

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by divyank00** [comment]
@devansh-299 Is this good?

File: app/src/main/java/org/mifos/mobile/ui/fragments/ThirdPartyTransferFragment.java [hunk 1]
```diff
@@ -39,6 +26,20 @@
 import org.mifos.mobile.utils.Toaster;
 import org.mifos.mobile.utils.Utils;
 
+import android.os.Bundle;
```
Comment (devansh-299) [inline app/src/main/java/org/mifos/mobile/ui/fragments/ThirdPartyTransferFragment.java:29]: Why remove the import lines and  add these lines again here?
Please update the PR , everything else seems fine to me :)

Comment (divyank00) [reply app/src/main/java/org/mifos/mobile/ui/fragments/ThirdPartyTransferFragment.java:29]: I didnt get this?

Comment (devansh-299) [reply app/src/main/java/org/mifos/mobile/ui/fragments/ThirdPartyTransferFragment.java:29]: You just changed the placement of the import statements, this is not necessary

**Comment by laxyapahuja** [comment]
This PR also contains commits from other issues. I think you'll need to close all pull requests and reset your branches and then do the changes.

**Review by ShivangiSingh17 — COMMENTED** [review]
Close the PR as it contains previously committed changes.

**Pull request closed without merging**